docs/docker-compose.md

```diff
@@ -0,0 +1,210 @@
+# Run with Docker Compose
+
+For more information about using Docker, see the [Docker Docs](https://docs.docker.com/). Make
+sure that you are using the latest version of Docker. The ones
+provided via `apt` and `yum` may be outdated and cause errors.
+
+We gather [Telemetry data] in the Percona packages and Docker images.
+
+--8<--- "get-help-snip.md"
+
+This guide shows you how to deploy a three-node Percona XtraDB Cluster 8.4 using Docker Compose. You generate SSL certificates on the first node and copy them to the other two nodes to enable secure communication.
+
+!!! note
+
+  This setup is for testing and development purposes only. Do not use it in production without configuring proper storage, security, backup, and monitoring systems.
+
+## Prerequisites
+
+* Docker and Docker Compose installed
+
+* At least 3 GB of memory per container
+	
+* Familiarity with Docker volumes and networks
+
+## Directory Structure
+
+You need to create a separate directory structure to organize your configuration, certificate files, and Docker Compose setup. This keeps your deployment clean and easy to manage.
+
+Run the following commands to create the directory structure:
+
+```{.bash data-prompt="$"}
+$ mkdir -p pxc-cluster/{certs,conf.d,init}
+$ cd pxc-cluster
+```
+
+After running these commands, your working directory (pxc-cluster/) will contain:
+
+![PXC cluster directories](_static/pxc-cluster-dirs.png)
+
+This structure helps manage configuration files, TLS/SSL certificates, and setup scripts, ensuring a tidy and easy-to-manage deployment.
+{.power-number}
+
+1. Create conf.d/custom.cnf with minimal SSL settings:
+  
+    ```ini
+    [mysqld]
+    ssl-ca=/etc/mysql/certs/ca.pem
+    ssl-cert=/etc/mysql/certs/server-cert.pem
+    ssl-key=/etc/mysql/certs/server-key.pem
+    ```
+
+2. Create a file named `.env` in the directory root:
+
+    ```ini
+    MYSQL_ROOT_PASSWORD=rootpass
+    XTRABACKUP_PASSWORD=xbpass
+    ```
+    
+    Add `.env` to your .gitignore file to prevent committing secrets to version control.
+
+3. Create the SSL Certificate Script. Save the following script as init/create-ssl-certs.sh:
+
+  ```ini
+  #!/bin/bash
+  set -e
+  
+  CERT_DIR=./certs
+  mkdir -p "$CERT_DIR"
+  cd "$CERT_DIR"
+  
+  openssl genrsa 2048 > ca-key.pem
+  openssl req -new -x509 -nodes -days 3650 \
+      -key ca-key.pem \
+      -subj "/C=XX/ST=State/L=City/O=Organization/CN=RootCA" \
+      -out ca.pem
+  
+  openssl req -newkey rsa:2048 -days 3650 -nodes \
+      -keyout server-key.pem \
+      -subj "/C=XX/ST=State/L=City/O=Organization/CN=pxc-node" \
+      -out server-req.pem
+  
+  openssl rsa -in server-key.pem -out server-key.pem
+  openssl x509 -req -in server-req.pem -days 3650 \
+      -CA ca.pem -CAkey ca-key.pem -set_serial 01 \
+      -out server-cert.pem
+ 
+
+  chmod 600 *.pem
+  ```
```

Comment:
The numbered list is broken after step 3

https://percona-xtradb-cluster-docs-8-4-pr-261.onrender.com/docker-compose.html#directory-structure